sdk/redis/Azure.ResourceManager.Redis/tests/ScenarioTests/GetListKeysFunctionalTests.cs

```diff
@@ -69,8 +69,8 @@ public async Task CreateUpdateDeleteTest()
 
             // Won't be equal when recording but might be equal in playback as all key values will be set to "Sanitized"
             // Make sure to manually edit session records so tests pass
-            Assert.AreNotEqual(response.PrimaryKey, afterRegenerateResponse.PrimaryKey);
-            Assert.AreNotEqual(response.SecondaryKey, afterRegenerateResponse.SecondaryKey);
+            Assert.AreEqual(response.PrimaryKey, afterRegenerateResponse.PrimaryKey);
```

Comment:
Based on the comment, should the session records be updated manually to reflect the actual test? Or should this test and comment be removed if we don't want to manually update after every recording?